index.js

```diff
@@ -0,0 +1,36 @@
+const ltClient = require("@lambdatest/node-rest-client");
```

Comment:
Let's rename the file to `LTresults.js` as index.js will collide with other packages.

Reply:
@akshatlambdatest index.js is specific to this package. Each package has it own index.js so it won't colide.
Also index.js conventionally gives sense of entry point that would be missing if we rename it to LTresult.js

Reply:
Defered